packages/main/src/types/ItemCheckMode.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Menu item group check modes.
+ * @since 2.11.0
+ * @public
+ */
+enum ItemCheckMode {
+	/**
+	 * default type (items in a group cannot be checked)
```

Comment:
Could you review which enumeration should remain? Having both `ItemCheckMode` and `ItemSelectionMode` seems a bit redundant to me.

Reply:
Yep! I forgot to delete the old one. Deleted.